apps/judicial-system/backend/src/app/modules/police/police.service.ts

```diff
@@ -247,6 +248,94 @@ export class PoliceService {
       })
   }
 
+  async getPoliceCaseInfo(
+    caseId: string,
+    user: User,
+  ): Promise<PoliceCaseInfo[]> {
+    const promise = this.fetchPoliceDocumentApi(
+      `${this.xRoadPath}/V2/GetDocumentListById/${caseId}`,
+    )
+
+    return promise
+      .then(async (res: Response) => {
+        if (res.ok) {
+          const response = await res
+            .json()
+            .then((result) =>
```

Comment:
Please consider removing this conditional second JSON parse. The api has been fixed, so we no longer need to handle this here.